src/simulator/simulator.py

```diff
@@ -58,14 +63,14 @@ def restart_func() -> None:
             if event.type == pygame.MOUSEBUTTONDOWN:
                 [b.click() for b in buttons]
 
-
         # fill the screen with a color to wipe away anything from last frame
         screen.fill("grey")
 
         render_buttons(screen, buttons)
 
         # optionally render nodes and edges. for now always on
-        render_world(screen, nodes, edges)
+        if route_visible:
+            render_world(screen, nodes, edges)
```

Comment:
render_world may hold other components in the future. Consider passing a flag through render_world to toggle the routes specifically.